Protecs/src/test/java/uk/ac/ucl/protecs/helperFunctions/helperFunctions.java

```diff
@@ -52,6 +52,20 @@ public static void runSimulationForTicks(WorldBankCovid19Sim sim, int numTicks)
 			sim.schedule.step(sim);
 		}
 	}	
+		
+	public static void SetFractionInfectionsWithCertainNode(double fraction, WorldBankCovid19Sim world, BehaviourNode Node) {
```

Comment:
This works great right now but might eventually become a problem where populations which already have infections are passed in, messing with the overall fraction of infections with a particular node. Because it's a utility function, it will (hopefully!) be reused a lot - which could cause trouble down the line